mesonbuild/backend/ninjabackend.py

```diff
@@ -1274,10 +1274,10 @@ def generate_dynamic_link_rules(self, outfile):
 '''
                 else:
                     command_template = ' command = %s %s $ARGS  %s $in $LINK_ARGS $aliasing\n'
-                command = command_template % \
-                (' '.join(compiler.get_linker_exelist()),\
-                 ' '.join(cross_args),\
-                 ' '.join(compiler.get_linker_output_args('$out')))
+                command = (command_template %
+                    (' '.join(compiler.get_linker_exelist()),
```

Comment:
__[flake8]__
* [E128] continuation line under-indented for visual indent

<!-- Comment by [SideCI](https://sideci.com) -->